src/stores/wordpress-versions-slice.ts

```diff
@@ -61,9 +61,9 @@ export const fetchWordPressVersions = createAsyncThunk(
 );
 
 interface WordPressVersion {
-	version: string;
 	isBeta: boolean;
-	name: string;
+	label: string;
+	value: string;
```

Comment:
Heads up, I'm renaming version and name to label and value for easy consumption in SelectControl.